subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/TestRetryPluginSmokeTest.groovy

```diff
@@ -20,8 +20,10 @@ package org.gradle.smoketests
 import org.gradle.test.fixtures.file.TestFile
 import org.gradle.testkit.runner.BuildResult
 import org.gradle.testkit.runner.TaskOutcome
+import spock.lang.Ignore
 import spock.lang.Issue
 
+@Ignore("This test is expected to fail. Exposing test failure types requires a breaking change in TestResultProcessor which is used by the test retry plugin")
```

Comment:
To keep track of this I added a TODO to the PR's description.

Reply:
Consider using `@PendingFeature` instead of just ignoring it.